webpack.config.js

```diff
@@ -70,6 +88,10 @@ module.exports = (env, argv) => {
       new MiniCssExtractPlugin({
         filename: devMode ? "[name]/assets/index.css" : "[name]/assets/index.[hash].css"
       }),
+      new HtmlWebpackPlugin({
```

Comment:
Without the `chunks: []` is the plugin injecting all the js/css files from all the entry points into the root index.html file?

Reply:
You're right, thanks, I pushed fix directly to master:
https://github.com/concord-consortium/vortex/commit/43fc85084c0a34234f08f0c2496f28adbbcf6cbe